xlsx.cabal

```diff
@@ -76,8 +77,14 @@ Library
                    , Codec.Xlsx.Writer
                    , Codec.Xlsx.Writer.Internal
                    , Codec.Xlsx.Writer.Internal.PivotTable
+                   , Codec.Xlsx.Parser.Stream
+                   , Codec.Xlsx.Writer.Stream
+                   , Codec.Xlsx.Writer.Internal.Stream
+
+  Other-modules:     Codec.Xlsx.Parser.Stream.HexpatInternal
```

Comment:
I think because the only function it exports is also hidden by the upstream library: https://github.com/the-real-blackh/hexpat/blob/master/Text/XML/Expat/SAX.hs#L227

We could expose it but then this function is in the xlsx API for a long time. It be better to expose it in the upstream library instead I think. It was copied here so the parser can use it.

Reply:
Sounds good, let's have a comment about this then?